src/test/java/de/tum/in/www1/artemis/assessment/GradingScaleUtilService.java

```diff
@@ -88,6 +108,16 @@ public GradingScale generateAndSaveGradingScale(int gradeStepCount, double[] int
         return gradingScaleRepository.save(gradingScale);
     }
 
+    /**
+     * Creates a grading scale with the given parameters.
```

Comment:
Here I think you should also write about the `fail("Invalid grading scale parameters")` in case of invalid values.